src/t_zset.c

```diff
@@ -763,7 +763,7 @@ sds lpGetObject(unsigned char *sptr) {
 }
```

Comment:
lpGetObject should be zlpGetObject?

Reply:
Yes，`lpGetObject()`  is changed from `ziplistGetObject()`, this should be named as `zlpGetObject`